package.json

```diff
@@ -42,6 +41,7 @@
     "moment": "^2.29.1",
     "mustache": "^4.2.0",
     "objects-to-csv": "^1.3.6",
+    "oclif": "^2",
```

Comment:
Is `oclif` necessary outside of `devDependencies`? Its functionality for us is replacing `oclif-dev` to do things such as call `yarn prepack` or `yarn version` in the scripts below.

Reply:
if it's not being used for anything else, then yeah let's move it to devdependencies